.github/workflows/master-pr-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: unit tests
+      env:
+        DEFAULT_BRANCH: master
+        GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      run: |
+        npm install
+        npm run lint
+        npm test -- --watch=false --progress=false --browsers=ChromeHeadlessCI
+
+  e2e-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: e2e tests
+      env:
+        DEFAULT_BRANCH: master
+        GH_TOKEN: ${{ secrets.GH_TOKEN }}
+      run: |
+        npm install
+        npm run lint
```

Comment:
since lint is in the 1. job I don't think it's required here

Reply:
Sure, second lint removed